data_steward/analytics/cdr_ops/ac120_cpt_modifiers.py

```diff
@@ -172,7 +172,26 @@ def cstr(s, color='black'):
 
 procedure_concept
 
-# # h) How would you fix the issue?
+# # h) What is the domain and concept of the affected concept ids
+
+# +
+#list procedure_concept_id's and counts to help determine if these records procedure_concept_id's are valid and if
+#these records should be updated rather than deleted
+
+domain = '''
+
+SELECT DISTINCT c.domain_id, standard_concept
```

Comment:
Please let me know when you've drafted a cleaning rule to address records containing cpt modifier codes. While our rule will focus on addressing only the records with cpt modifier codes, it may be that these are just a subset of coding issues that could be discovered using a query like below.
```
SELECT 
 c.domain_id,
 c.concept_class_id,
 c.standard_concept,
 COUNT(1) n
FROM `{DATASET_ID}.procedure_occurrence` p
JOIN {DATASET_ID}.concept c 
 ON p.procedure_source_concept_id = c.concept_id
GROUP BY c.domain_id, c.concept_class_id, c.standard_concept
ORDER BY n DESC, c.standard_concept, c.concept_class_id, c.domain_id
```